### ISSUE TITLE
Changes to  keyboard help hint

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/resistance-in-a-wire#135]
After an interview @terracoda said she would like to update the hint for keyboard help content.  @terracoda when this is done can you please let me know?

Once this is done we will likely be ready to deploy an an RC of this sim with a11y description.

**Comment by jessegreenberg** [comment]
Duplicate of #136,  closing.
